sql/core/src/test/scala/org/apache/spark/sql/execution/ExchangeSuite.scala

```diff
@@ -132,4 +132,12 @@ class ExchangeSuite extends SparkPlanTest with SharedSQLContext {
     val projection2 = cached.select("_1", "_3").queryExecution.executedPlan
     assert(!projection1.sameResult(projection2))
   }
+
+  test("SPARK-26601: Make broadcast-exchange thread pool configurable") {
+
+    withSQLConf(StaticSQLConf.MAX_BROADCAST_EXCHANGE_THREADNUMBER.key -> "1") {
+      assert(SQLConf.get.getConf(StaticSQLConf.MAX_BROADCAST_EXCHANGE_THREADNUMBER) === 1)
+      assert(BroadcastExchangeExec.executionContext != null)
```

Comment:
`BroadcastExchangeExec.executionContext` can't be null. Looks it doesn;t properly test the change.

Reply:
ok @HyukjinKwon i will try to add more cases